client/jetpack-connect/authorize.js

```diff
@@ -261,6 +266,14 @@ export class JetpackAuthorize extends Component {
 			window.close();
 		}
 
+		const urlParams = new URLSearchParams( window.location.search );
+		const source = urlParams.get( 'source' );
+
+		if ( source === 'jetpack-manage' ) {
+			const urlRedirect = `${ JPC_JETPACK_MANAGE_PATH }?site_connected=${ urlToSlug( homeUrl ) }`;
```

Comment:
Maybe we can use `addQueryArgs` to build this URL?

Reply:
Yes, I agree! I have made the changes.